Tools/machines/frontier-olcf/submit.sh

```diff
@@ -45,5 +45,61 @@ export ROCFFT_RTC_CACHE_PATH=/dev/null
 export OMP_NUM_THREADS=1
 export WARPX_NMPI_PER_NODE=8
 export TOTAL_NMPI=$(( ${SLURM_JOB_NUM_NODES} * ${WARPX_NMPI_PER_NODE} ))
-srun -N${SLURM_JOB_NUM_NODES} -n${TOTAL_NMPI} --ntasks-per-node=${WARPX_NMPI_PER_NODE} \
-    ./warpx inputs > output.txt
+export WARPX_OUTPUT=output.txt
```

Comment:
Maybe let's include the job id to avoid overwriting on restarts?
```suggestion
export WARPX_OUTPUT=output_${SLURM_JOB_ID}.txt
```

Reply:
I mean, I guess that's up to the user if they need restart? This is supposed to be a sample and I don't want it to be too complicated, but also this is not necessarily an issue to add